Realm/RLMRealm.h

```diff
@@ -695,6 +695,18 @@ NS_REFINED_FOR_SWIFT;
  */
 - (void)deleteAllObjects;
 
+#pragma mark - Sync Subscriptions
+
+/**
+ Returns an instance of `SyncSubscriptionSet`, representing the active subscriptions
+ for this realm, which can be used to add/remove/update and search flexible sync subscriptions.
+ Getting the subscriptions from a local or partition-based configured realm will return a nil subscription
+ set.
+
+ @return A `RLMSyncSubscriptionSet`.
+ */
```

Comment:
This is written as documentation for a method which returns a value rather than a property.